src/contracts/contractManager.ts

```diff
@@ -1,20 +1,19 @@
 import type { AssetId, ChainId } from '@shapeshiftoss/caip'
 import { fromAssetId, toAssetId } from '@shapeshiftoss/caip'
-import type { EvmChainId } from '@shapeshiftoss/chain-adapters'
 import type { Token } from '@uniswap/sdk'
 import { Fetcher } from '@uniswap/sdk'
-import {
-  ERC20ABI__factory,
-  FarmingAbi__factory,
-  IUniswapV2Pair__factory,
-  IUniswapV2Router02__factory,
-  THORCHAIN_Router__factory,
-} from 'contracts/__generated'
+import { FarmingABI } from 'contracts/abis/farmingAbi'
+import { IUniswapV2Pair } from 'contracts/abis/IUniswapV2Pair'
+import { IUniswapV2Router02 } from 'contracts/abis/IUniswapV2Router02'
+import { THORChain_RouterABI } from 'contracts/abis/THORCHAIN_RouterABI'
 import { ethers } from 'ethers'
 import memoize from 'lodash/memoize'
+import type { Address } from 'viem'
+import { getContract } from 'viem'
 import { getEthersProvider } from 'lib/ethersProviderSingleton'
+import { viemEthMainnetClient } from 'lib/viem-client'
 
-import type { IUniswapV2Pair } from './__generated'
+import { erc20ABI } from './abis/ERC20ABI'
```

Comment:
throw this import up alongside the rest of the `contracts/abis` imports?

Reply:
https://github.com/shapeshift/web/pull/5532/commits/b50bb9ddcfa4ef403a906deb978dd41e91732c7d